lib/internal/bootstrap/switches/is_not_main_thread.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-const { ObjectDefineProperty } = primordials;
+const {
```

Comment:
Can we not enforce multi lines?

Reply:
It's to minimize the chance of conflicts – git might be able to merge two multiline declarations, but if they are monoline it would require manual intervention. Any reason not to use multiline?